tests/unit_tests/services/test_dds.py

```diff
@@ -386,27 +386,23 @@ def test_release_project_nomail(self):
             ])
 
```

Comment:
The `test_get_dds_project_title` function has been removed, is this intentional?

Reply:
Thank you for noting this. That was a big mistake from my side. I guess I copied that test to duplicate it and ended up editing it instead.